index.js

```diff
@@ -3,73 +3,98 @@ $("#selectOption").val(localStorage.getItem("optionValue"));
+  }
+}
 
-    const mappedUsers = arrOfObjects.map((user)=>{
-        return getMappedUser(user);
+function draw(arrOfObjects) {
+  const mappedUsers = arrOfObjects.map((user) => {
+    return getMappedUser(user);
+  });
+  
+//   //DRAW
+    cardsContainer.empty();
+    const cards = mappedUsers.map((user) => {
+        return drawCards(user)
     })
-    console.log(mappedUsers);
+}
 
-    //DRAW
-};
+function drawCards(user) {
+    const mainDiv = $(`<div></div>`).addClass("card").css({width: "18rem", display: "inline-block"});
+    const cardBody = $("<div></div>").addClass("card-body");
+    const userImg = $(`<img src="${user.imgSrc}">`);
+    const userName = $(`<h6><b>Name:</b> ${user.name}</h6>`).addClass("card-title");
+    const userCity = $(`<p><b>City:</b> ${user.city}</p>`).addClass("card-title");
+    const userAddress = $(`<p><b>Address:</b> ${user.address}</p>`).addClass("card-title");
 
-function getMappedUser (user) {
-    const keyValueMappingArray = Object.entries(mapping);
-    return keyValueMappingArray.reduce((mappedUser, KEYVALUEPAIR_ARRAY,) => {
-        const [ key, settingObj ] = KEYVALUEPAIR_ARRAY;
-        const { path } = settingObj;
-        const isFunction = typeof settingObj[ "fn" ] === 'function';
-        return { ...mappedUser, [ key ]: isFunction ? settingObj[ "fn" ](user) : getValueFromPath(path, user) };
-    }, {});
-};
+    mainDiv.append(userImg)
+    mainDiv.append(cardBody)
+    cardBody.append(userName)
+    cardBody.append(userCity)
+    cardBody.append(userAddress)
+    cardsContainer.append(mainDiv)
 
-function getValueFromPath (path, user) {
-    if (typeof path !== 'string') return;
-    const splittedPath = path.split(".");
-    const theRequestedValue = splittedPath.reduce((currentUser, partOfPath) => {
-        const isValueExist = currentUser[ partOfPath ];
-        return isValueExist ? currentUser[ partOfPath ] : "Not Availble";
-    }, user);
-    return theRequestedValue;
-};
+    return cardsContainer;
+}
 
+function getMappedUser(user) {
+  const keyValueMappingArray = Object.entries(mapping);
+  return keyValueMappingArray.reduce((mappedUser, KEYVALUEPAIR_ARRAY) => {
+    const [key, settingObj] = KEYVALUEPAIR_ARRAY;
+    const { path } = settingObj;
+    const isFunction = typeof settingObj["fn"] === "function";
+    return {
+      ...mappedUser,
+      [key]: isFunction ? settingObj["fn"](user) : getValueFromPath(path, user),
+    };
+  }, {});
+}
 
+function getValueFromPath(path, user) {
+  if (typeof path !== "string") return;
+  const splittedPath = path.split(".");
+  const theRequestedValue = splittedPath.reduce((currentUser, partOfPath) => {
+    const isValueExist = currentUser[partOfPath];
+    return isValueExist ? currentUser[partOfPath] : "Not Availble";
+  }, user);
+  return theRequestedValue;
+}
 
-$("#selectOption").on("change",()=>{
-    init();
-    window.localStorage.setItem("optionValue", $("#selectOption").val());
```

Comment:
i would suggest to create a local storage adapter
function getFromLS(key,isParsed){

}

function setLocalStorage(key,value,isParsed){

}